lib/plugins/aws/deploy/compile/events/sns/index.js

```diff
@@ -51,11 +51,11 @@ class AwsCompileSNSEvents {
             }
 
             const lambdaLogicalId = this.provider.naming
-              .getLogicalLambdaName(functionName);
+              .getLambdaLogicalId(functionName);
             const topicLogicalId = this.provider.naming
-              .getLogicalSnsTopicName(topicName);
+              .getTopicLogicalId(topicName);
```

Comment:
imo we shouldn't have to be too picky or have a specific standard for methods names here, otherwise we'd need another `naming.js` file to centralize the method names in this `naming.js` file 😅 ... This is super clear that it generates the topic logical ID